README.md

```diff
@@ -289,6 +289,7 @@ If you want to add a new loader, for example, to process `json` files via webpac
 
 ```
 yarn add json-loader
+npm install --save json-loader
```

Comment:
That's only true if `save=false` isn't in an `.npmrc` file somewhere, as many of us do who don't like npm 5+'s default autosave behavior. I think it's a good idea to keep it here.